faasmcli/requirements.txt

```diff
@@ -1,5 +1,6 @@
 ansible
 awscli
+black
```

Comment:
I just realized that we installed black as part of `grpc-root` [here](https://github.com/faasm/faabric/blob/0b863976bc8b88ed904cc1ae89dce9a63a514685/docker/grpc-root.dockerfile#L22).